### ISSUE TITLE
Using JW Player with VoiceOver is impossible

### ISSUE THREAD

**Issue by francoismassart** [jwplayer/jwplayer#1183]
### Expected Behavior

We would like to be able to use JW Player with iOS and Mac OS' VoiceOver...
The different buttons of the player should be labelled (and translated too) for the screen reader softwares.

### Actual Behavior

The buttons of the player are empty and only contain an icon injected via CSS

### User story

We received a complaint from a blind visitor.

He is using Apple's VoiceOver screen reader in order to surf to our sites on his Mac and on his iPhone and he is having difficulties to play the videos.

We are using  on our site auvio:
http://www.rtbf.be/auvio/detail_?id=2106569

Or even easier, you can simply open the src of the iframe containing the video player with this url
http://www.rtbf.be/auvio/embed/media?id=2106569&autoplay=0

When I try it by myself using my iPhone 6S with VoiceOver, sliding my finger blindly accross the screen, I don't get much useful VoiceOver description when I'm touching the video...

I hear "Afficher les contrôles, bouton" ("Show the controls, button") when I'm near the bottom area of the video area...
The big play button is generated with this markup:

`<div class="jw-icon jw-icon-display jw-button-color jw-reset"></div>`

This markup does not contains any text consummable by the screen readers such as the Twitter Bootstrap utility class like

`<span class="sr-only">Play the video</span>`

or a similar aria attribute...

What can we do in order to improve the VoiceOver experience knowing that the markup is generated by the JW Player js script ?

Thank you very much

### Environment

- MacBook Pro and iPhone 6S
- Mac OS Yosemite and iOS 9
- JWPlayer v7.3.6 with ads

----------------------------------------------------------------------------------------

This template is provided as a guide to filing bugs and feature requests. Please replace 
all text *in italics* with details describing your issue. When submitting a feature request, 
"Steps to reproduce" can be substituted with a user story or list of acceptance criteria.

**Comment by fdambrosio** [comment]
we have same problem too with blind user.

**Comment by robwalch** [comment]
While we have made it a priority to support FCC compliant captions (608, 708, positioning), we do not have other accessibility features like adding aria attributes the UI scheduled in our backlog.

Please submit requests to our support team for this feature. It really helps when our support and product teams have this on their radar. Or submit a PR. Most of the UI elements in the player have HTML templates which are pretty easy to modify. For example:

https://github.com/jwplayer/jwplayer/blob/master/src/templates/displayicon.html

If the main focus is attributes on buttons in the controlbar, that can be modified here:

https://github.com/jwplayer/jwplayer/blob/master/src/js/view/controlbar.js#L15

**Comment by francoismassart** [comment]
My P.R. only add attributes but the tooltip's overlay and the sliders (e.g. HD submenu, timeline, volume slider...) do not work using the keyboard because they use only the `mouseover` event to expand.
That could be improved later.
